### PR TITLE
Update homepage

### LINKED ISSUE #3: Home page looks blank
### Describe your use-case which is not covered by existing documentation.

The home page for the documentation looks empty. I would suggest adding at least some description.

image

@krisstern @Cr1t-GYM 

### Reference any relevant documentation, other materials or issues/pull requests that can be used for inspiration.

_No response_

### PR DESCRIPTION
**Pull request by Cr1t-GYM**
- [x] Update home page.
- [x] Add logo for all pages.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -10,3 +10,14 @@ permalink: /
 
 ---
 **Hello Jenkinsfile-runner actions!**
+
+This is the project page of Jenkinsfile-runner Actions.
+Jenkinsfile-runner Actions intends to design suitable GitHub Actions for users so Jenkins pipeline can be run in the GitHub Actions' runners.
```
Comment (krisstern) [inline index.md]:
```suggestion
Jenkinsfile-runner Actions intends to be a design suitable for GitHub Actions so the Jenkins pipeline can be run in the GitHub Actions' runners.
```

File: index.md [hunk 2]
```diff
@@ -10,3 +10,14 @@ permalink: /
...
+
+To launch the pipeline, you only need to provided the Jenkinsfile and simply quote our actions in your workflow definition.
+The pipeline results can be viewed in the GitHub pages.
+You can also configure your ephemeral Jenkins instances by providing JCasC file, prebuilt container urls or combine with other GitHub Actions!
```
Comment (krisstern) [inline index.md]:
```suggestion
You can also configure your ephemeral Jenkins instances by providing a suitable JCasC file or a set of prebuilt container urls, or by combining with other GitHub Actions.
```

**Review by krisstern — CHANGES_REQUESTED** [review]

**Review by krisstern — APPROVED** [review]

**Review by AbhyudayaSharma — APPROVED** [review]

**Comment by Cr1t-GYM** [comment]
This is weird. It doesn't show the logo like the local testing.

**Comment by krisstern** [comment]
I think for the favicon you may need to add something similar to the following in the `<head></head>` section of your layout:
```html
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">

    <link rel="shortcut icon" href="{{ 'favicon.ico' | relative_url }}" type="image/x-icon">
</head>
```

Like [this](https://github.com/physics-vault/physics-vault.github.io/blob/gh-pages/_layouts/head.html).

**Comment by krisstern** [comment]
For the logo on the pages, it looks like this to me now:
image

**Comment by Cr1t-GYM** [comment]
> For the logo on the pages, it looks like this to me now: image

Great! I don't know why I cannot see the logo several minutes before. But it does work now.

**Comment by krisstern** [comment]
Awesome! No worries

**Pull request merged**